docs/en/papers/stam-cn.md

```diff
@@ -0,0 +1,160 @@
+STAM：针对大型分布式应用系统的拓扑自动检测方法
```

Comment:
This file path is weird, `docs/en/papers/stam-cn.md`, it contains `en` and `cn`

Reply:
Removed CN version.